roles/hammer_devel/tasks/hammer_config.yml

```diff
@@ -57,11 +57,17 @@
     creates: ~/.hammer/cli.modules.d/katello.yml
   when: ('katello/hammer-cli-katello' in hammer_devel_repositories)
 
-- name: 'Configure hammer-cli-csv'
-  command: cp ~/hammer-cli-csv/config/csv.yml ~/.hammer/cli.modules.d/
+- name: 'Configure hammer-cli-azure'
```

Comment:
Isn't there some way to iterate in a loop here?

Reply:
I can try to refactor it so it looks cleaner with all the tasks